rebar.config

```diff
@@ -151,12 +151,12 @@
  [
   %% This plugin dowloads zips from github and caches them
   %% This plugin is totally optional and everything would work without it (just slower)
-  {rebar_faster_deps, {git, "https://github.com/arcusfelis/rebar3-faster-deps-plugin.git",
-      {ref, "425c75af169f40b9c1c9e471b941175c4530374f"}}},
-  {pc, {git, "https://github.com/blt/port_compiler.git", {ref, "c2f3fb1"}}},
-  {provider_asn1, {git, "https://github.com/arcusfelis/provider_asn1.git", {ref, "29f7850"}}},
+  %{rebar_faster_deps, {git, "https://github.com/arcusfelis/rebar3-faster-deps-plugin.git",
+  %    {ref, "425c75af169f40b9c1c9e471b941175c4530374f"}}},
+  {pc, "1.11.0"},
+  {provider_asn1, "0.2.3"},
   {rebar3_codecov, {git, "https://github.com/zofpolkowska/rebar3_codecov.git", {ref, "b539c5c"}}},
```

Comment:
Why aren't we using the ESL one?
https://github.com/esl/rebar3_codecov

Reply:
I was simply not aware of or didn't remember about `esl/rebar3_codecove`. I'll try it.